src/humanPose/MuriScore.js

```diff
@@ -0,0 +1,818 @@
+import {MotionStudyColors} from "./MotionStudyColors.js";
+import {ErgonomicsData, clamp} from "./ErgonomicsData.js";
+import {JOINT_CONNECTIONS, JOINTS, getBoneName, ERGO_ANGLES, ERGO_OFFSETS} from './constants.js';
+
+/** Default configuration of thresholds for MURI calculation and visualisation. */
+const MURI_CONFIG_DEFAULT = Object.freeze({
+    trunkFrontBendAngleThresholds: [-5, 20, 45],  // degrees ; normally [0, 20, 45] but we give margin of -5 deg to accommodate imperfect accuracy of pose when upright 
+    trunkSideBendAngleThresholds: [15, 30],  // degrees
+    trunkTwistAngleThresholds: [15, 45],  // degrees
+    headFrontBendAngleThresholds: [-3, 15, 30],  // degrees ; normally [0, 15, 30] but we give margin of -3 deg to accommodate imperfect accuracy of pose when upright 
+    headSideBendAngleThresholds: [15, 30],  // degrees
+    headTwistAngleThresholds: [15, 30],  // degrees
+    footHeightDifferenceThresholds: [150], // mm   
+    lowerLegBendAngleThresholds: [60],  // degrees
+    upperArmFrontRaiseAngleThresholds: [60, 90], // degrees
+    upperArmSideRaiseAngleThresholds: [60, 90], // degrees
+    lowerArmBendAngleThresholds: [120], // degrees
+    lowerArmTwistAngleThresholds: [120],  // degrees
+    handFrontBendAngleThresholds: [-20, 30], // degrees
+    handReachDistanceThresholds: [300, 500],  // mm   //TODO: tune based on customer feedback
+
+    // Weigths for different levels of strain on a body part which are used to calculate actual score.
+    // Note: Weights are integer numbers in the latest iteration of MURI method
+    scoreWeights: [0, 1, 2, 5], 
```

Comment:
Where do these come from?

Reply:
These are score weights defined in Volvo's MURI Excel sheet for different levels of severity.